.vib/postgresql/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/postgresql"
         },
-        "runtime_parameters": "YXV0aDoKICB1c2VybmFtZTogdXNlcgogIHBhc3N3b3JkOiBDb21wbGljYXRlZFBhc3N3b3JkMTIzNAogIGRhdGFiYXNlOiB0ZXN0X2RhdGFiYXNlCnByaW1hcnk6CiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBwb3N0Z3Jlc3FsOiA4MAogICAgdHlwZTogTG9hZEJhbGFuY2Vy",
+        "runtime_parameters": "YXJjaGl0ZWN0dXJlOiByZXBsaWNhdGlvbgphdXRoOgogIGVuYWJsZVBvc3RncmVzVXNlcjogdHJ1ZQogIHBvc3RncmVzUGFzc3dvcmQ6ICJDb21wbGljYXRlZFBhc3N3b3JkMTIzITQiCiAgdXNlcm5hbWU6IHRlc3RfdXNlcgogIHBhc3N3b3JkOiB0ZXN0X3Bhc3N3b3JkCiAgZGF0YWJhc2U6IHRlc3RfZGF0YWJhc2UKICByZXBsaWNhdGlvblVzZXJuYW1lOiB0ZXN0X3JlcGxfdXNlcgpjb250YWluZXJQb3J0czoKICBwb3N0Z3Jlc3FsOiA1NDMzCnBvc3RncmVzcWxTaGFyZWRQcmVsb2FkTGlicmFyaWVzOiAicGdhdWRpdCwgcGdfc3RhdF9zdGF0ZW1lbnRzIgpzaG1Wb2x1bWU6CiAgZW5hYmxlZDogdHJ1ZQpwcmltYXJ5OgogIG5hbWU6IHB0ZXN0CiAgcG9kU2VjdXJpdHlDb250ZXh0OgogICAgZW5hYmxlZDogdHJ1ZQogICAgZnNHcm91cDogMTAwMgogIGNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICAgIGVuYWJsZWQ6IHRydWUKICAgIHJ1bkFzVXNlcjogMTAwMgogIHNlcnZpY2U6CiAgICB0eXBlOiBMb2FkQmFsYW5jZXIKICAgIHBvcnRzOgogICAgICBwb3N0Z3Jlc3FsOiA1NDMzCiAgcGVyc2lzdGVuY2U6CiAgICBlbmFibGVkOiB0cnVlCiAgICBtb3VudFBhdGg6IC9iaXRuYW1pL3Bvc3RncmVzcWwKcmVhZFJlcGxpY2FzOgogIG5hbWU6IHJ0ZXN0CiAgcmVwbGljYUNvdW50OiAyCiAgcG9kU2VjdXJpdHlDb250ZXh0OgogICAgZW5hYmxlZDogdHJ1ZQogICAgZnNHcm91cDogMTAwMgogIGNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICAgIGVuYWJsZWQ6IHRydWUKICAgIHJ1bkFzVXNlcjogMTAwMgogIHNlcnZpY2U6CiAgICB0eXBlOiBMb2FkQmFsYW5jZXIKICAgIHBvcnRzOgogICAgICBwb3N0Z3Jlc3FsOiA4MAogIHBlcnNpc3RlbmNlOgogICAgZW5hYmxlZDogdHJ1ZQogICAgbW91bnRQYXRoOiAvYml0bmFtaS9wb3N0Z3Jlc3Fs",
```

Comment:
No strong thoughts on this, but is it necessary to have `readReplicas.persistence.mountPath` defined here? We are maintaining its default value and changes to the persistence conf shouldn't affect our testing environment.

Reply:
I think @joancafom followed the rule of _all we can be customized works_

Reply:
Yes, that was the rule I followed. However, we discussed it internally and I changed it to only include things that we can either assert in the tests to be working or that directly affect the test environment